week-1/K-functions-parameters/exercise5.js

```diff
@@ -1,3 +1,6 @@
+function createLongGreeting(string, num) {
```

Comment:
Just a tiny comment, that it's more readable if you give your parameters meaningful names, in this case `name` and `age` would be more readable :)